### ISSUE TITLE
The date picker icon suggests it is disabled on the listing page

### ISSUE THREAD

**Issue by LieutenantRoger** [topcoder-platform/micro-frontends-challenges-app#227]
**Summary:**
Contrast of the calendar icon in the 'from' input box does not comply with WCAG standards. Also it should be greeen (CTA colored).

**Steps to Reproduce:**

1. Open https://platform.topcoder.com/earn/find/challenges
2. Open the date selector

**Actual Result:**
The contrast between the background and the icon is lover than the minimum standard.

**Expected Result:**
The contrast should be higher

**Environment Details:**

***Devices(s):***
MacBook Pro
***Operating System and version:***
MacOS Catalina
***Browser and version:***
Safari 14.0.3
**Screen recording or Screen snapshot:**
![Screenshot_2021-10-29_at_17.18.31](/uploads/4873da43109244a157b7b4d8819c48fd/Screenshot_2021-10-29_at_17.18.31.png)

**Comment by LieutenantRoger** [comment]
Seems it's fine

**Comment by Oanh-and-only-Oanh** [comment]
not an issue
